### PR TITLE
[subset] Rename codepoint -> unicode in subset plan api.

### LINKED ISSUE #3461: [subset] Rename hb_subset_plan_codepoint_to_old_glyph_mapping
Codepoint should be renamed to unicode?

### PR DESCRIPTION
**Pull request by garretrieger**
Fixes #3461

### CODE REVIEW

**Comment by behdad** [comment]
Change in the docs/ dir please as well.

**Comment by behdad** [comment]
Please rebase on main.

**Comment by garretrieger** [comment]
> Please rebase on main.

Done.

File: docs/harfbuzz-sections.txt [hunk 1]
```diff
@@ -777,4 +777,13 @@ hb_subset_plan_codepoint_to_old_glyph_mapping
...
 hb_subset_or_fail
+hb_subset_plan_execute_or_fail
+hb_subset_plan_create_or_fail
+hb_subset_plan_destroy
+hb_subset_plan_old_to_new_glyph_mapping
+hb_subset_plan_new_to_old_glyph_mapping
+hb_subset_plan_unicode_to_old_glyph_mapping
+hb_subset_plan_reference
+hb_subset_plan_set_user_data
+hb_subset_plan_get_user_data
```
Comment (khaledhosny) [inline docs/harfbuzz-sections.txt:788]: These entries are already in the file, you only need to rename hb_subset_plan_codepoint_to_old_glyph_mapping.

Comment (garretrieger) [reply docs/harfbuzz-sections.txt]: Fixed

**Pull request merged**